plugins.json

```diff
@@ -489,6 +489,14 @@
         "entry_point_prefix": "dynamic_workflows",
         "code_home": "https://github.com/microsoft/aiida-dynamic-workflows"
     },
+    "aiida-champ": {
+    "entry_point_prefix": "champ",
+    "development_status": "beta",
+    "plugin_info": "https://raw.github.com/TREX-CoE/aiida-champ/master/setup.json",
+    "code_home": "https://github.com/TREX-CoE/aiida-champ",
+    "pip_url": "aiida-champ"
```

Comment:
```suggestion
    "pip_url": "aiida-champ",
```

Reply:
thanks, this edit is necessary for a valid JSON

(somehow I'm not able to commit to your PR so you'll need to do it ;-) )